File: backend/Api/Parkings/Contracts/ParkingResponse.cs
using JetBrains.Annotations;

namespace Api.Parkings.Contracts;

[PublicAPI]
public sealed record ParkingResponse
{
    public required Guid Id { get; init; }
    public required string Code { get; init; }
    public required string Name { get; init; }
    public required string Address { get; init; }
    public required int SpotsCount { get; init; }
    public required string OwnerId { get; init; }
}

File: mobile/endpoints/parkings/parking-response.ts
export type ParkingResponse = {
  readonly id: string;
  readonly address: string;
  readonly name: string;
  readonly spotsCount: number;
  readonly ownerId: string;
  readonly code: string;
  readonly isFull: boolean;
  readonly maxSpots: number;
  readonly isNeighbourhood: boolean;
};
